libs/ledger-live-common/src/env.ts

```diff
@@ -46,6 +46,11 @@ const jsonParser = (v: unknown): JSONValue | undefined => {
   }
 };
 
+const stringArrayParser = (v: any): string[] | null | undefined => {
```

Comment:
Function can't return undefined

Reply:
Why not forcing the type of `v` to be `string` here ?

Reply:
Part of the code that I got from the previous PR, but with the way it's coded right now, yes it can return `undefined`.
```typescript
if (Array.isArray(v_array) && v_array.length > 0) return v_array;
// else return undefined
```

for the `v` type, `envDefinitions` is typed as `Record<EnvName, EnvDef<any>>` where `EnvDef` is:
```
type EnvDef<V> = {
  desc: string;
  def: V;
  parser: (arg0: unknown) => V | null | undefined;
};
``` 
Meaning that at this stage, the typing is poor, but without changing everything (since it's not related to this PR) `v` can be either `unknown` or `any`.